README.md

```diff
@@ -13,43 +13,62 @@ install_github("epiverse-trace/datadelay")
 library(datadelay)
```

Comment:
This .md file seems different to the newer .Rmd it should be generated from? Does it just need to be refreshed?

Reply:
Yes, the rendering isn't turned on for this branch - can turn it on for all branches

Reply:
The `.md` file updates automatically from the `.Rmd`, fixed in https://github.com/epiverse-trace/datadelay/pull/11/commits/a68473618a3e77408c56e3f9ec4011975c58f000